phpmd.xml

```diff
@@ -12,7 +12,24 @@
   <rule ref="rulesets/design.xml">
     <exclude name="CouplingBetweenObjects"/>
   </rule>
-  <rule ref="rulesets/naming.xml"/>
+
+  <rule ref="rulesets/naming.xml">
+    <exclude name="ShortVariable"/>
+    <exclude name="LongVariable"/>
+  </rule>
+
+  <rule ref="rulesets/naming.xml/ShortVariable">
+    <properties>
+        <property name="exceptions" value="id,op"/>
+    </properties>
+  </rule>
+
+  <rule ref="rulesets/naming.xml/LongVariable">
+    <properties>
+        <property name="maximum" value="30"/>
+    </properties>
+  </rule>
+
```

Comment:
Didn't we already merge all this? If not, let's quickly merge this in another PR.

Reply:
These are already merged. I will rebase with main.

Reply:
Ahh sorry. there are new updates in DQC, I will raise a separate PR.